configs/kde/root-image/var/lib/AccountsService/users/antergos

```diff
@@ -1,5 +1,5 @@
 [User]
 Language=en_US.UTF-8
-XSession=kde
+XSession=plasma.desktop
```

Comment:
@Wyn10 I don't think that it should have `.desktop` included in the session name. Check what we are using for new installs in Cnchi code to be sure.